extensions-core/kinesis-indexing-service/src/test/resources/base64aggregatedkinesismessage.txt

```diff
@@ -0,0 +1 @@
OS0wMS0yNVQxMDoyNjozNS42NjAtMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6NzEuODgxNzUwODE0OTM0MjJ9Gm8IABpreyJ0aW1lc3RhbXAiOiIyMDE5LTAxLTI1VDEwOjI2OjM1LjY2NS0wODowMCIsIm15U3RyaW5nIjoic29tZSB0ZXh0IiwibXlJbnQiOjQsIm15RG91YmxlIjo3Ny41MjExMTEyNjM0NTE3Nn0abwgAGmt7InRpbWVzdGFtcCI6IjIwMTktMDEtMjVUMTA6MjY6MzUuNjYwLTA4OjAwIiwibXlTdHJpbmciOiJzb21lIHRleHQiLCJteUludCI6NCwibXlEb3VibGUiOjM1LjAzOTExMjI0ODM4MTkyfRptCAAaaXsidGltZXN0YW1wIjoiMjAxOS0wMS0yNVQxMDoyNjozNS42NjUtMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6ODEuMDY0MzAxODU0MzIyfRpuCAAaansidGltZXN0YW1wIjoiMjAxOS0wMS0yNVQxMDoyNjozNS42NjEtMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6MzUuMTU1NjE1MjY1OTI4NH0acAgAGmx7InRpbWVzdGFtcCI6IjIwMTktMDEtMjVUMTA6MjY6MzUuNjY3LTA4OjAwIiwibXlTdHJpbmciOiJzb21lIHRleHQiLCJteUludCI6NCwibXlEb3VibGUiOjU3LjM2MzkwODA3Mjc3MjQ5Nn0abwgAGmt7InRpbWVzdGFtcCI6IjIwMTktMDEtMjVUMTA6MjY6MzUuNjY3LTA4OjAwIiwibXlTdHJpbmciOiJzb21lIHRleHQiLCJteUludCI6NCwibXlEb3VibGUiOjgyLjI4NzgzMjM5NTkyNjM0fRpvCAAaa3sidGltZXN0YW1wIjoiMjAxOS0wMS0yNVQxMDoyNjozNS42NjgtMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6ODcuNjU5NDQ4MjYyNzkzNDl9GnAIABpseyJ0aW1lc3RhbXAiOiIyMDE5LTAxLTI1VDEwOjI2OjM1LjY2OC0wODowMCIsIm15U3RyaW5nIjoic29tZSB0ZXh0IiwibXlJbnQiOjQsIm15RG91YmxlIjoxMi43NTUyNjUxMTQwNzg4ODV9Gm4IABpqeyJ0aW1lc3RhbXAiOiIyMDE5LTAxLTI1VDEwOjI2OjM1LjY2OC0wODowMCIsIm15U3RyaW5nIjoic29tZSB0ZXh0IiwibXlJbnQiOjQsIm15RG91YmxlIjo4Mi4wODI5NTEyODA0MDEzfRpwCAAabHsidGltZXN0YW1wIjoiMjAxOS0wMS0yNVQxMDoyNjozNS42NjktMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6NC44NTgzMjQyMzg4NzIyNTg1fRpvCAAaa3sidGltZXN0YW1wIjoiMjAxOS0wMS0yNVQxMDoyNjozNS42NjktMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6NTAuMzE4OTc5NDM0NTA4NTZ9Gm8IABpreyJ0aW1lc3RhbXAiOiIyMDE5LTAxLTI1VDEwOjI2OjM1LjY3MC0wODowMCIsIm15U3RyaW5nIjoic29tZSB0ZXh0IiwibXlJbnQiOjQsIm15RG91YmxlIjozNi45NjY5MDkyODE0NTI0OX0abwgAGmt7InRpbWVzdGFtcCI6IjIwMTktMDEtMjVUMTA6MjY6MzUuNjcwLTA4OjAwIiwibXlTdHJpbmciOiJzb21lIHRleHQiLCJteUludCI6NCwibXlEb3VibGUiOjk2LjgzMzM5MzE4Mjc0NDI3fRpuCAAaansidGltZXN0YW1wIjoiMjAxOS0wMS0yNVQxMDoyNjozNS42NzAtMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6NDUuMTQyMDcwMjkxNTA2MX0abggAGmp7InRpbWVzdGFtcCI6IjIwMTktMDEtMjVUMTA6MjY6MzUuNjcxLTA4OjAwIiwibXlTdHJpbmciOiJzb21lIHRleHQiLCJteUludCI6NCwibXlEb3VibGUiOjcwLjg0MjE5MjY3MjM5Mzl9Gm8IABpreyJ0aW1lc3RhbXAiOiIyMDE5LTAxLTI1VDEwOjI2OjM1LjY3MS0wODowMCIsIm15U3RyaW5nIjoic29tZSB0ZXh0IiwibXlJbnQiOjQsIm15RG91YmxlIjo2Ni40NDU2NDU2ODE2MzkyNX0abwgAGmt7InRpbWVzdGFtcCI6IjIwMTktMDEtMjVUMTA6MjY6MzUuNjcxLTA4OjAwIiwibXlTdHJpbmciOiJzb21lIHRleHQiLCJteUludCI6NCwibXlEb3VibGUiOjUuMDY0MzczODg3MzcxNzMxfRpvCAAaa3sidGltZXN0YW1wIjoiMjAxOS0wMS0yNVQxMDoyNjozNS42NzEtMDg6MDAiLCJteVN0cmluZyI6InNvbWUgdGV4dCIsIm15SW50Ijo0LCJteURvdWJsZSI6NjMuNTQ1MjE5MjM3ODc2MDd9Gm8IABpreyJ0aW1lc3RhbXAiOiIyMDE5LTAxLTI1VDEwOjI2OjM1LjY3Mi0wODowMCIsIm15U3RyaW5nIjoic29tZSB0ZXh0IiwibXlJbnQiOjQsIm15RG91YmxlIjo1My4zNDI4ODM3MjcwNDg5M31foE8ypcllp3vZXupl8YPE
```

Comment:
Does this file need a license?  If so, I'll have to add something to the reader code in tests to ignore the header

Reply:
It shouldn't, we have a rat exclusion for `<exclude>**/test/resources/**</exclude>`